test/integration/testhelper.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func AssertResponseStatus(assert *assert.Assertions, url string, statusCode int) {
+	var resStatusCode int
+	resStatusCode, _, _ = httpGetRequest(url)
+	assert.Equal(statusCode, resStatusCode)
+}
+
+func httpGetRequest(url string) (statusCode int, body string, err error) {
```

Comment:
nit: Let's make this func take `t` and in case of errors below

```suggestion
func httpGetRequest(url string) (t *testing.T, statusCode int, body string, err error) {
  t.Helper()
```